projects/vdk-plugins/vdk-jobs-troubleshooting/src/vdk/plugin/jobs_troubleshoot/api/troubleshoot_utility.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from abc import abstractmethod
+
+
+class ITroubleshootUtility:
+    """
+    Allows for troubleshoot utilities to be implemented. It provides a unified
+    API to be used by the jobs troubleshooting plugin irrespective of the
+    utility's underlying logic.
+
+    A troubleshooting utility is a utility class, containing logic which is to
+    be used for troubleshooting of deployed data jobs.
+    """
+
+    @abstractmethod
+    def start(self):
+        """
+        Starts the execution of the utility when the job is initialized.
```

Comment:
You may mention where is the [VDK job cycle](https://github.com/vmware/versatile-data-kit/tree/main/projects/vdk-plugins#types-of-plugins) is start and stop called , perhaps?

Reply:
Updated doc. Thanks